dataikuapi/dss/projectdeployer.py

```diff
@@ -0,0 +1,450 @@
+from .future import DSSFuture
```

Comment:
Is it on purpose that we don't expose:
* `com.dataiku.dip.server.api.PublicAPIProjectDeployerController#deleteProject`
* `com.dataiku.dip.server.api.PublicAPIProjectDeployerController#listStages` 

?

Reply:
For now I merely followed the methods we had in `apideployer.py`, "translating" them to Project deployer methods. For `deleteProject` especially, there was no endpoint in the Public API to delete services before dataiku/dip#11101. We can consider adding those to both deployers? 🙂

Reply:
Done in 6a220bd